tests/auth.spec.js

```diff
@@ -13,15 +13,27 @@ const { userMock } = mockData;
 const { forgotPasswordEmail, wrongForgotPasswordEmail } = userMock;
 
 const BASE_URL = '/api/v1';
-const endpoint = `${BASE_URL}/users`;
+const signupEndpoint = `${BASE_URL}/users`;
+const loginsignupEndpoint = `${BASE_URL}/users/login`;
 const FORGOT_PASSWORD_URL = `${BASE_URL}/auth/forgotPassword`;
 
+
+before((done) => {
```

Comment:
I suggest you place this `before` hook in the describe block of the login route you need it to test since it's not needed by the other tests in this file. Another approach you can take is to seed a user into the database and test the login route by logging in that user.